src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs

```diff
@@ -231,5 +235,105 @@ public Task Disable()
 
             return NoContent();
         }
+
+        [RavenAction("/databases/*/admin/indexes/dump", "POST", AuthorizationStatus.DatabaseAdmin)]
+        public Task Dump()
+        {
+            var name = GetStringQueryString("name");
+            var path = GetStringQueryString("path");
+            var index = Database.IndexStore.GetIndex(name);
+            if (index == null)
+            {
+                IndexDoesNotExistException.ThrowFor(name);
+                return null;//never hit
+            }
+
+            var operationId = Database.Operations.GetNextOperationId();
+            var token = CreateTimeLimitedQueryOperationToken();
+            var details = new DumpIndexOperationDetails
+            {
+                Index = name,
+            };
+
+            var task = Database.Operations.AddOperation(
+                Database,
+                "Dump index " + path,
```

Comment:
Was this supposed to be `"Dump index " + name`, or maybe even `"Dump index " + name + " to" + path` ?

Reply:
Yes, fixed